scalableminds/webknossos
# Newrelic support in docker

Description of changes:
- Adds the newrelic agent to the docker installation
- Newrelic will only be activated if `NEW_RELIC_LICENSE_KEY` is present as env variable

Steps to test:
- Run `docker-compose up oxalis` normally
- Add `NEW_RELIC_LICENSE_KEY` as environment variable
- Run `docker-compose up oxalis` again
- Check the logs

Issues:
- fixes #1547 

------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1551/create?referer=github" target="_blank">Log Time</a>
